tests/test_triton_server.py

```diff
@@ -42,7 +42,7 @@
 }
 
 
-class TestTritonServerMethods(trc.TestResultCollector):
+class TestTritonServerMethods(unittest.TestCase):  #trc.TestResultCollector):
```

Comment:
Why did you remove trc?

Reply:
For some reason I was having trouble getting the unittests to work on my machine with the trc.